src/users/components/invite-user-permissions/invite-user-permission.component.html

```diff
@@ -1,31 +1,47 @@
-    [formGroupName]="'roles'"
-    [error]="errorMessages">
+    <lib-gov-uk-form-group-wrapper
+      [config]="{hint: 'Choose at least one action the user can do. You can change this later.', legend: 'Permissions', key: 'roles'}"
+      [formGroupName]="'roles'"
+      [error]="errorMessages">
 
-    <div class="govuk-checkboxes">
-      <lib-gov-checkbox
-        [group]="inviteUserForm.controls['roles']"
-        [isChecked]="isPuiCaseManager"
-        [config]="{value: 'pui-case-manager',  label: 'Manage Cases', name: 'roles', hint: 'View, create and progress cases.', focusOn: 'roles'}">
-      </lib-gov-checkbox>
+      <div class="govuk-checkboxes">
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiCaseManager"
+          [config]="{value: 'pui-case-manager',  label: 'Manage cases', name: 'roles', hint: 'View, create and progress cases.', focusOn: 'roles'}">
+        </lib-gov-checkbox>
 
-      <lib-gov-checkbox
-        [group]="inviteUserForm.controls['roles']"
-        [isChecked]="isPuiUserManager"
-        [config]="{value: 'pui-user-manager', label: 'Manage Users', name: 'roles',
-        hint: 'Invite users.'}">
-      </lib-gov-checkbox>
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiUserManager"
+          [config]="{value: 'pui-user-manager', label: 'Manage users', name: 'roles',
+          hint: 'Invite users.'}">
+        </lib-gov-checkbox>
 
-      <lib-gov-checkbox
-        [group]="inviteUserForm.controls['roles']"
-        [isChecked]="isPuiOrganisationManager"
-        [config]="{value: 'pui-organisation-manager', label: 'Manage Organisation', name: 'roles',
-        hint: 'View organisation name and addresses.'}">
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiOrganisationManager"
+          [config]="{value: 'pui-organisation-manager', label: 'Manage organisation', name: 'roles',
+          hint: 'View the organisation name and addresses.'}">
+        </lib-gov-checkbox>
+
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isCaseAccessAdmin"
+          [config]="{value: 'pui-caa', label: 'Case access administrator', name: 'roles',
+          hint: 'Manage case access for all users within your organisation. You can assign this role to any user in your organisation. However, your service might not currently support this role. You will receive further guidance once it becomes available.'}">
       </lib-gov-checkbox>
+        <!-- <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiFinanceManager"
+          [config]="{value: 'pui-finance-manager', label: 'Manage Payments', name: 'roles',
+          hint: 'View account balances, available credit and transactions.'}">
+        </lib-gov-checkbox> -->
```

Comment:
remove this if comment out

Reply:
agreed. if im not allowed to leave commented code, neither can you lol